python/tvm/relay/tensorrt.py

```diff
@@ -123,6 +162,16 @@ def _func_wrapper(attrs, args):
     return _func_wrapper
 
 
+def _register_external_dynamic_check_func(op_name, func):
```

Comment:
It might be better to register this as a decorator so we don't have to call `_register_external_dynamic_check_func` everytime. Not required though